gto-support-db/src/androidTest/java/org/ccci/gto/android/common/db/ExpressionTest.java

```diff
@@ -0,0 +1,51 @@
+package org.ccci.gto.android.common.db;
+
+import android.support.test.runner.AndroidJUnit4;
+import android.test.InstrumentationTestCase;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.ccci.gto.android.common.db.TestDao.TestContract.RootTable;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+@RunWith(AndroidJUnit4.class)
+public class ExpressionTest extends InstrumentationTestCase {
+
+    private TestDao getDao() {
+        return new TestDao(getInstrumentation().getContext());
+    }
+
+    @Test
+    public void testEqualsSql() {
+        final TestDao dao = getDao();
+
+        Expression equalsExpression = RootTable.FIELD_TEST.eq("1");
+        assertThat(equalsExpression.buildSql(dao).first, is("(root.test == ?)"));
+    }
+
+    @Test
+    public void testNotEqualsSql() {
+        final TestDao dao = getDao();
+
+        Expression notEqualsExpression = RootTable.FIELD_TEST.not().eq("1");
```

Comment:
so, while this test is valid, this isn't something that should probably happen in normal usage. the expression you are probably looking for is: `FIELD_TEST.eq("1").not()` or `FIELD_TEST.ne("1")`